pyproject.toml

```
@@ -0,0 +1,21 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "example"
+version = "0.1.0"
+requires-python = ">=3.10"
+
+[project.dependencies]
+numpy = "^1.21.0"
+matplotlib = "^3.4.0"
+
```

```suggestion
  [project]
  dependencies = [
      "numpy>=1.20"
      "matplotlib>=3.4.0"
  ]
```

---

I added several other libraries for data-processing

```suggestion
[project.dependencies]
numpy = "^1.21.0"
matplotlib = "^3.4.0"
pandas = "2.2.0"
json = "1.6.3"
  ]
```